installer/pkg/config/v1/config.go

```diff
@@ -81,7 +81,7 @@ type Config struct {
 
 	Workspace Workspace `json:"workspace" validate:"required"`
 
-	AuthProviders []AuthProviderConfigs `json:"authProviders"`
+	AuthProviders []AuthProviderConfigs `json:"authProviders" validate:"dive"`
```

Comment:
Cool, looks like this will "dive" into the next level (each member) and trigger its related validator.

Reply:
Correct. We don't need to make `AuthProviders` required (if not set, it'll allow users to use the GUI version of this). The `dive` property [tells it to interrogate the slice](https://github.com/go-playground/validator/blob/master/_examples/simple/main.go#L16)

Reply:
Perfect, thank you!